### DISCUSSION TITLE
Can't load subscriber data from csv file when running Docker container

### DISCUSSION THREAD

**Discussion by pyroteck** [srsran/srsRAN_Project#488] [General Help] [answered]
When trying to load subscriber data via a csv file when running the Docker container, I get an error saying "No such file or directory"

```
open5gs_5gc     | SUBSCRIBER_DB=subscriber_db.csv
open5gs_5gc     | Reading subscriber data from csv-file.
open5gs_5gc     | Error reading subscriber_db.csv: [Errno 2] No such file or directory: 'subscriber_db.csv'
open5gs_5gc     | Failed to add subscribers to database
```

My `open5gs.env` file is edited as follows:
```
MONGODB_IP=127.0.0.1
OPEN5GS_IP=10.53.1.2
UE_IP_BASE=10.45.0
DEBUG=false
SUBSCRIBER_DB="subscriber_db.csv"
```

I'm following the guide outlined [here](https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html#id17) where it says "The file needs to be saved at: srsRAN_Project/docker/open5gs/", which for some reason in the README located in the [docker folder directory](https://github.com/srsran/srsRAN_Project/tree/main/docker) shows "It must be stored in srsgnb/docker/open5gs/", but I've searched every directory on my machine and there's no instance of an `srsgnb/` directory. I've tried replacing "subscriber_db.csv" with "/home/{usr}/srsRAN_Project/docker/open5gs/subscriber_db.csv" and I still get the same error output.

**Comment by pgawlowicz** [comment, marked as answer]
Hi, 
do you start the docker container as follows:
```
cd ./srsRAN_Project/docker
docker compose up --build 5gc
```

i.e., from `./srsRAN_Project/docker` directory?   The README file has to be fixed as there is no `srsgnb` directory.

**Reply by pyroteck** [reply]
Yes, I do start it from that directory, and I didn't change anything in my project but it seems to be working now after running `sudo docker compose up --build 5gc`. Before I was doing `sudo docker compose -f docker-compose.yml up` as I need Grafana as well, but that also seems to be working fine now.

**Reply by pgawlowicz** [reply]
once you change any file in the docker directory, you need to rebuild, so the changed files are used to build new images. I guess that now you can use :
```
sudo docker compose up
```

**Reply by pyroteck** [reply]
Ah okay that makes sense, I'm new to anything involving containerization so I didn't know. I see now that the README mentions adding the `--build` flag for forcing a new build but didn't think to try adding it. `sudo docker compose up` does work within the `./srsRAN_Project/docker` directory now.
